extensions/guacamole-auth-ldap/src/main/java/org/apache/guacamole/auth/ldap/user/UserService.java

```diff
@@ -85,11 +85,22 @@ private void putAllUsers(Map<String, User> users, LDAPConnection ldapConnection,
 
         try {
 
+            // Build a filter using the configured or default user search filter
+            // to find all user objects in the LDAP tree
+            StringBuilder userSearchFilter = new StringBuilder();
+            userSearchFilter.append("(&");
+            userSearchFilter.append(confService.getUserSearchFilter());
+            userSearchFilter.append("(");
+            userSearchFilter.append(escapingService.escapeLDAPSearchFilter(usernameAttribute));
+            userSearchFilter.append("=*)");
+            userSearchFilter.append(")");
+         
+
```

Comment:
Please kill one of these blank lines - separating things by two blanks is somewhat unusual compared to the surrounding code.